community/modules/internal/slurm-gcp/instance_template/files/startup_sh_unlinted

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 
 set -e
+set -o xtrace
```

Comment:
This file is called `unlinted` to workaround shell-linter that forced us to make changes to this file content.
We wanted to avoid such changes as it would force replacement of templates. I see few options here:
* Embrace breaking change: rename it and fix all lint error + your changes;
* Lazily embrace breaking change: what you have in your PR (totally fine);
* Make it non-breaking: create a separate startup-script for TPUs, keep non-TPU as-is.

WDYT?

Reply:
Went with:
> Make it non-breaking: create a separate startup-script for TPUs, keep non-TPU as-is.